api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -4161,6 +4174,9 @@
     <concept_set concept_set_id="2182" concept_id="90315" concept_set="163169" sort_weight="2.0" creator="1" date_created="2016-07-28 13:26:18.0" uuid="2181AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
     <concept_set concept_set_id="2183" concept_id="163172" concept_set="163169" sort_weight="2.0" creator="1" date_created="2016-07-28 13:26:18.0" uuid="2182AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
     <concept_set concept_set_id="2184" concept_id="1305" concept_set="163157" sort_weight="2.0" creator="1" date_created="2016-07-28 13:26:18.0" uuid="2184AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
+    <concept_set concept_set_id="2404" concept_id="1305" concept_set="163151" sort_weight="2.0"  creator="52"  date_created="2017-11-14 13:07:48"  uuid="2404AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
```

Comment:
@slubwama This is the creator who was causing issues for me - double check and ensure that all creators have a value of 1